sdk/src/types.ts

```diff
@@ -2,14 +2,13 @@
 // Modifications Copyright (c) 2025 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+import { Network } from '@iota/iota-sdk/dist/cjs/client/network.js';
```

Comment:
Doesn't this work?
```suggestion
import { Network } from '@iota/iota-sdk/client';
```
`dist/cjs/` should definitely not be there

Reply:
yeah not sure what happened